susemanager/src/mgr_sync/mgr_sync.py

```diff
@@ -42,9 +42,16 @@ class MgrSync(object):  # pylint: disable=too-few-public-methods
 
     def __init__(self):
         self.config = Config()
-        url = "http://{0}:{1}{2}".format(self.config.host,
-                                         self.config.port,
-                                         self.config.uri)
+
+        # Use http on localhost by default to avoid hairpins when running in kubernetes
+        ssl = ""
```

Comment:
I would suggest to change the code a bit this way if you don't mind:
```suggestion
        scheme = "http"
        if self.config.host not in ["localhost", "127.0.0.1"]:
            scheme = "https"
            self.config.port = 443
        url = "{0}://{1}:{2}{3}".format(
            scheme,
            self.config.host,
            self.config.port,
            self.config.uri,
        )
```

Reply:
Thanks for the hint, applied.